libraries/crypto/src/ecdsa.rs

```diff
@@ -140,7 +142,7 @@ impl SecKey {
         }
     }
 
-    pub fn from_bytes(bytes: &[u8; 32]) -> Option<SecKey> {
+    pub fn from_bytes(bytes: &[u8; NBYTES]) -> Option<SecKey> {
```

Comment:
I don't think this change makes it clearer on the public API of this module. And changes in this file are unrelated to this pull request anyway.

Reply:
I can revert the last commit. Please agree with @kaczmarczyck .

Reply:
Go ahead and merge with the 32, and we can do library changes outside of this PR if we still think we want them.

Reply:
SGTM. Reverted.